packages/utils/src/lib/reports/formatting.ts

```diff
@@ -58,3 +66,58 @@ export function metaDescription(
   }
   return '';
 }
+
+/**
+ * Link to local source for IDE
+ * @param source
+ * @param reportLocation
+ *
+ * @example
+ * linkToLocalSourceInIde({ file: '/src/index.ts'}, {outputDir: '/.code-pushup'}) // [/src/index.ts](../src/index.ts)
+ */
+export function linkToLocalSourceForIde(
+  source: SourceFileLocation,
+  options?: Pick<MdReportOptions, 'outputDir'>,
+): InlineText {
+  const { file, position } = source;
+  const { outputDir } = options ?? {};
+  const unixPath = toUnixPath(file);
```

Comment:
It will already be in Unix format, the [`formatGitPath`](https://github.com/code-pushup/cli/blob/main/packages/utils/src/lib/git/git.ts#L10-L14) function takes care of that.

Reply:
The Windows tests kept failing without this conversion (see [here](https://github.com/code-pushup/cli/actions/runs/11040734885/job/30669397342)). Do you think the test itself was incorrect and it wasn't the code problem?

Reply:
Oh. Well, then I guess the tests input was in Windows format? You could always call `toUnixPath` in the test to prepare the input if need be.

Reply:
The `linkToLocalSourceForIde` test input uses forward slashes, but when running tests for Windows, it gets internally changed to the Windows-style path format. As a result, when tested in isolation, it didn't produce the expected results.

I am not very familiar with the code base, so I added `toUnixPath` as a precaution. If you confirm that it will work as expected without `toUnixPath`, then I will add it to the test instead.

Reply:
Yes, you can assume that `issue.source.file` was been normalized to relative Git path.